### ISSUE TITLE
Linking Levels (to create a full Metroidvania World)

### ISSUE THREAD

**Issue by TheYellowArchitect** [TheYellowArchitect/doubledamnation#2]
Levels have no limits, yet, the bigger a level gets, the laggier it gets to load.
What if you want to make a world as big as the first level, or even greater? A full metroidvania world?
This is possible with this feature.
		
In short, in every level exit/ending, you can attach a level ID, so instead of finishing and going to the [level browser](https://github.com/TheYellowArchitect/doubledamnation/issues/13), you instantly load that level.
I will tell you why this is hype. Because look at any metroidvania world design. There are often max 3 entrances to a new area. Often there is just one entrance, and a second is harder to reach or is the exit of that area.
Look at Symphony of the Night for example.
![00-CD](https://user-images.githubusercontent.com/25630803/184157973-5857d97b-050b-4a98-bd6d-8cc02b8765c1.png)
These areas are literally the level links of this here.
		
tl;dr: Link levels to create your own metroidvania world without any limit.
		
"B-But the loading time!"
The downloading of the linked level should not start when you get to the exit, but start when you get X range from it.
And ofc it is cached in your pc and instantly loaded once you get inside.

**This is the evolution of the level editor.**
